docs/pages/admin-guides/management/security/selinux.mdx

```diff
@@ -118,7 +113,7 @@ The module source consists of two files, the type enforcement file and the file
 
 The type enforcement file is the heart of the module and contains statements that define what Teleport SSH processes are allowed to do.
 
-The file contexts file maps files and directories Teleport SSH uses to the appropriate SELinux labels. For more information on SELinux labels refer to the Red Hat article [SELinux Contexts - Labeling Files](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/7/html/selinux_users_and_administrators_guide/sect-security-enhanced_linux-working_with_selinux-selinux_contexts_labeling_files).
+The file contexts file maps files and directories Teleport SSH uses to the appropriate SELinux labels. For more information on SELinux labels refer to the Red Hat article [SELinux Contexts - Labeling Files](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/selinux_users_and_administrators_guide/sect-security-enhanced_linux-working_with_selinux-selinux_contexts_labeling_files).
```

Comment:
Does "Teleport SSH" refer to the Teleport SSH Service? If so, I'd use the latter term for consistency with the rest of the docs.

Reply:
done